src/parser/command-line.ts

```diff
@@ -1,28 +1 @@
-import * as github from "@actions/github";
-import { config } from "dotenv";
-import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as WebhookEventName } from "@octokit/webhooks";
-
-config();
-
 export type SupportedEvents = "issues.closed";
```

Comment:
I think we can find this type some friends to live with instead of sitting here alone